README.md

```diff
@@ -24,6 +24,7 @@ If you think you’ve found a potential security issue, please do not post it in
 * `experimental_concurrency_retries`: Specify a limit to the number of retries concurrent goroutines will attempt.  By default `4` retries will be attempted before records are dropped.
 * `aggregation`: Setting `aggregation` to `true` will enable KPL aggregation of records sent to Kinesis.  This feature isn't compatible with the `partition_key` feature.  See the KPL aggregation section below for more details.
 * `compression`: Setting `compression` to `zlib` will enable zlib compression of each record.  By default this feature is disabled and records are not compressed.
+* `replace_dots`: If you set replace_dots as true, all dots in key names will be replaced with underscores.
```

Comment:
So I just thought of something... this feature is too limited. You can just take dots and replace them with underscores. But what if you don't want underscores? What if you have a key like `example.key` and a key `example_key`- with this feature when the first one has its dot replaced it will then conflict with the second key.

So instead, let's have the value of the `replace_dots` be a string which will be what you replace the dots with.

So for example you can do:

```
[OUTPUT]
    Name            kinesis
    Match           *
    region          us-west-2
    stream          my-kinesis-stream-name
    aggregation     true
    append_newline  true
    replace_dots    -
```

This would replace dots with a `-`. Basically let the user decide what is the replacement. If they want underscores, they can specify it.

Reply:
Make sense. Will implement the option like this.